spring-data-relational/src/main/java/org/springframework/data/relational/core/mapping/schemasqlgeneration/ColumnModel.java

```diff
@@ -29,16 +29,16 @@ public class ColumnModel implements Serializable {
     @Serial
     private static final long serialVersionUID = 1L;
     private final SqlIdentifier name;
-    private final Class<?> type;
+    private final String type;
```

Comment:
Decided to simplify the model to be a String value.    Whatever is in here is what gets in the generated SQL.